src/api/java/com/minecolonies/api/colony/managers/interfaces/ICitizenManager.java

```diff
@@ -151,6 +157,12 @@ default ICitizenData spawnOrCreateCitizen(final ICitizenData data, @NotNull fina
      */
     void setMaxCitizens(final int newMaxCitizens);
 
+    /**
+     * Set the new potential max citizens.
+     * @param newMaxCitizens the potential amount to set.
+     */
+    void setPotentialMaxCitizens(final int newMaxCitizens);
```

Comment:
What is the difference between getMaxCitizens() and getPotentialMaxCitizens()?
Better documentation is needed, what Counts as a potential here?

Reply:
As a note, that says `set` not `get`, just incase that matters